src/scss/main.scss

```diff
@@ -111,17 +111,27 @@ body {
 }
 
 /* little shadow on top of the screen for no reasons */
-body::before {
+/* TinyMCE selector for body in preview excluded to prevent flashing */
+body::before,
+#tinymce-preview::before {
   box-shadow: 0 0 10px rgba(0, 0, 0, 0.8);
   content: '';
-  height: 10px;
   left: 0;
-  position: fixed;
   top: -10px;
   width: 100%;
+}
+
+body::before {
+  height: 10px;
+  position: fixed;
   z-index: 100;
 }
 
+#tinymce-preview::before {
```

Comment:
can it be a class instead of an id? There are pages with several tinymce editors.